prime-router/src/testIntegration/resources/datatests/HL7_to_FHIR/sample_co_1_20220518-0014.fhir

```diff
@@ -168,38 +163,34 @@
           } ]
         },
         "who" : {
-          "reference" : "Organization/1666796916944776000.f27b5ffc-4e5f-44aa-9736-d57ca90bcb11"
+          "reference" : "Organization/1671810652472721700.a2572421-9d1d-4156-a63a-d97f7495af09"
         }
       } ],
       "entity" : [ {
         "role" : "source",
         "what" : {
-          "reference" : "Device/1666796916951775200.f6f63b18-5f4f-417d-99e9-11c3e7b7cf79"
+          "reference" : "Device/1671810652479719200.120fb818-bde7-4b54-9c6c-a9723b4264bc"
         }
       } ]
     }
   }, {
-    "fullUrl" : "Organization/1666796916944776000.f27b5ffc-4e5f-44aa-9736-d57ca90bcb11",
+    "fullUrl" : "Organization/1671810652472721700.a2572421-9d1d-4156-a63a-d97f7495af09",
     "resource" : {
       "resourceType" : "Organization",
-      "id" : "1666796916944776000.f27b5ffc-4e5f-44aa-9736-d57ca90bcb11",
+      "id" : "1671810652472721700.a2572421-9d1d-4156-a63a-d97f7495af09",
       "meta" : {
         "extension" : [ {
-          "url" : "https://reportstream.cdc.gov/fhir/StructureDefinition/source-processing-id",
-          "valueCodeableConcept" : {
-            "coding" : [ {
-              "system" : "https://terminology.hl7.org/3.1.0/CodeSystem-v2-0103.html",
-              "code" : "P"
-            } ]
-          }
-        }, {
-          "url" : "http://ibm.com/fhir/cdm/StructureDefinition/source-record-id",
-          "valueId" : "MT_COCZS_ORU_ZSPHELR.1.6384446"
-        }, {
           "url" : "http://ibm.com/fhir/cdm/StructureDefinition/source-data-model-version",
           "valueString" : "2.5.1"
+        } ],
+        "tag" : [ {
+          "system" : "http://terminology.hl7.org/CodeSystem/v2-0103",
+          "code" : "P"
         } ]
       },
+      "identifier" : [ {
+        "value" : "COCZS"
+      } ],
       "name" : "COCZS",
```

Comment:
I noticed your latest commit added this change, but I'm having trouble wrapping my head around why. Not saying that it's wrong, but was that expected? What change caused this to be added here when it wasn't here before the last commit?

Reply:
This is a good catch.  The identifier is supposed to replace the name.  Looks like I did not regenerated this test file, but now it is gone.